code/modules/antagonists/cult/team_cult.dm

```diff
@@ -0,0 +1,485 @@
+	var/list/living_players = get_living_players(exclude_nonhuman = TRUE, exclude_offstation = TRUE)
+	var/players = length(living_players)
+	var/cultists = get_cultists() // Don't count the starting cultists towards the number of needed conversions
+	if(players >= CULT_POPULATION_THRESHOLD)
+		// Highpop
+		ascend_percent = CULT_ASCENDANT_HIGH
+		rise_number = round(CULT_RISEN_HIGH * (players - cultists))
+		ascend_number = round(CULT_ASCENDANT_HIGH * (players - cultists))
+	else
+		// Lowpop
+		ascend_percent = CULT_ASCENDANT_LOW
+		rise_number = round(CULT_RISEN_LOW * (players - cultists))
+		ascend_number = round(CULT_ASCENDANT_LOW * (players - cultists))
+
+/datum/team/cult/proc/study_objectives(mob/living/M, display_members = FALSE) //Called by cultists/cult constructs checking their objectives
+	if(!M)
+		return FALSE
+
+	switch(cult_status)
+		if(NARSIE_IS_ASLEEP)
+			to_chat(M, "<span class='cult'>[GET_CULT_DATA(entity_name, "The Dark One")] is asleep. This is probably a bug.</span>")
+		if(NARSIE_DEMANDS_SACRIFICE)
+			var/list/all_objectives = objective_holder.get_objectives()
+			if(!length(all_objectives))
+				to_chat(M, "<span class='danger'>Error: No objectives. Something went wrong, adminhelp with F1.</span>")
+			else
+				var/datum/objective/sacrifice/current_obj = all_objectives[length(all_objectives)] //get the last obj in the list, ie the current one
+				to_chat(M, "<span class='cult'>The Veil needs to be weakened before we are able to summon [GET_CULT_DATA(entity_title1, "The Dark One")].</span>")
+				to_chat(M, "<span class='cult'>Current goal: [current_obj.explanation_text]</span>")
+		if(NARSIE_NEEDS_SUMMONING)
+			to_chat(M, "<span class='cult'>The Veil is weak! We can summon [GET_CULT_DATA(entity_title3, "The Dark One")]!</span>")
+			to_chat(M, "<span class='cult'>Current goal: [obj_summon.explanation_text]</span>")
+		if(NARSIE_HAS_RISEN)
+			to_chat(M, "<span class='cultlarge'>\"I am here.\"</span>")
+			to_chat(M, "<span class='cult'>Current goal:</span> <span class='cultlarge'>\"Feed me.\"</span>")
+		if(NARSIE_HAS_FALLEN)
+			to_chat(M, "<span class='cultlarge'>[GET_CULT_DATA(entity_name, "The Dark One")] has been banished!</span>")
+			to_chat(M, "<span class='cult'>Current goal: Slaughter the unbelievers!</span>")
+		else
+			to_chat(M, "<span class='danger'>Error: Cult objective status currently unknown. Something went wrong, adminhelp with F1.</span>")
+
+	if(!display_members)
+		return
+	var/list/cult = get_cultists(separate = TRUE)
+	var/total_cult = cult[1] + cult[2]
+
+	var/overview = "<span class='cultitalic'><br><b>Current cult members: [total_cult]"
+	if(!cult_ascendant)
+		var/rise = rise_number - total_cult
+		var/ascend = ascend_number - total_cult
+		if(rise > 0)
+			overview += " | Conversions until Rise: [rise]"
+		else if(ascend > 0)
+			overview += " | Conversions until Ascension: [ascend]"
+	to_chat(M, "[overview]</b></span>")
+
+	if(cult[2]) // If there are any constructs, separate them out
```

Comment:
```suggestion
	if(length(cult[2])) // If there are any constructs, separate them out
```

Reply:
sadly its not a list, its an number
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/6501ea47-fa94-401e-b554-fe2ec54e872c)

Reply:
pain